Poliklinika.Model/obj/Poliklinika.Model.csproj.nuget.dgspec.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Moodel/obj treba stavit u gitignore jer su to stvari koje ce bit razlicite kod men i kod tebe